kore/src/Kore/Builtin/External.hs

```diff
@@ -44,94 +48,50 @@ externalize ::
     TermLike variable ->
     Syntax.Pattern variable Attribute.Null
 externalize =
-    Recursive.unfold worker
+    Recursive.futu externalize1
   where
-    worker ::
-        TermLike variable ->
-        Recursive.Base
-            (Syntax.Pattern variable Attribute.Null)
-            (TermLike variable)
-    worker termLike =
+    externalize1 ::
+        Futu (Syntax.Pattern variable Attribute.Null) (TermLike variable)
+    externalize1 termLike =
         -- TODO (thomas.tuegel): Make all these cases into classes.
         case termLikeF of
-            InternalBoolF (Const internalBool) ->
-                (toPatternF . Recursive.project) (Bool.asTermLike internalBool)
-            InternalIntF (Const internalInt) ->
-                (toPatternF . Recursive.project) (Int.asTermLike internalInt)
-            InternalBytesF (Const internalBytes) ->
-                (toPatternF . Recursive.project)
-                    (InternalBytes.asTermLike internalBytes)
-            InternalStringF (Const internalString) ->
-                (toPatternF . Recursive.project)
-                    (String.asTermLike internalString)
-            InternalListF internalList ->
-                (toPatternF . Recursive.project)
-                    (List.asTermLike internalList)
-            InternalMapF internalMap ->
-                (toPatternF . Recursive.project)
-                    (Map.asTermLike internalMap)
-            InternalSetF internalSet ->
-                (toPatternF . Recursive.project)
-                    (Set.asTermLike internalSet)
-            InjF inj ->
-                (toPatternF . Recursive.project . synthesize . ApplySymbolF)
-                    (Inj.toApplication inj)
-            _ -> toPatternF termLikeBase
+            AndF andF                 -> (Attribute.Null :<) $ Syntax.AndF         $ fmap Pure andF
```

Comment:
We should add `From` instances for all of the `Syntax.PatternF` branches.

Reply:
I found it better to just define a small local function since all of these are effectively the same and we don't really wanna spread this externalization code too far wide. Let me know what you think of this.